app/views/projects/_form.html.erb

```diff
@@ -1,26 +1,47 @@
 <%= form_for @project do |f| %>
-  <div>
-    <%= f.label :name %>
-    <%= f.text_field :name %>
+<div id="project">
+<div class="col-lg-6">
```

Comment:
I think it might improve the readability if you were to indent all of the HTML tags.